### ISSUE TITLE
TimeMixer to Multi task

### ISSUE THREAD

**Issue by TianQingX** [kwuking/TimeMixer#95]
How can I TimeMixer to Multi-task, Your work is not bad for short-term prediction, and I would like to join your experiment for multi-task vol prediction on finance

**Comment by TianQingX** [comment]
Means predict more than one time series target, like multi time series target(Y1,Y2,Y3) for next 60 steps



发自我的iPhone


------------------ Original ------------------
From: MetaKing ***@***.***&gt;
Date: Wed,Dec 4,2024 5:47 PM
To: kwuking/TimeMixer ***@***.***&gt;
Cc: TianQingXia ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [kwuking/TimeMixer] TimeMixer to Multi task (Issue #95)





 
Thank you for your interest in our work. May I ask what you mean by "multi-task"? TimeMixer is currently capable of performing short-term predictions directly—does this meet your requirements?

**Comment by kwuking** [comment]
Yes, this is an annotated multivariate time series medical data. TimeMixer can fully handle multivariate time series prediction. The test datasets currently provided in our scripts, such as weather, ETT, and Traffic, are all multivariate time series. You can refer to these settings and directly use TimeMixer to handle this problem.

**Comment by kwuking** [comment]
> Means predict more than one time series target, [Global forecasting models](https://cienciadedatos.net/documentos/py44-multi-series-forecasting-skforecast) , like multi time series target(Y1,Y2,Y3) for next 60 steps. for example, In weather predicting, but predicting temperature, Moist, wind speed

Yes, this is an annotated multivariate time series medical data. TimeMixer can fully handle multivariate time series prediction. The test datasets currently provided in our scripts, such as weather, ETT, and Traffic, are all multivariate time series. You can refer to these settings and directly use TimeMixer to handle this problem.
